core/trino-main/src/main/java/io/trino/server/InternalCommunicationConfig.java

```diff
@@ -31,7 +31,7 @@
 public class InternalCommunicationConfig
 {
     private String sharedSecret;
-    private boolean http2Enabled;
+    private boolean http2Enabled = true;
```

Comment:
The PR name should be `Enable HTTP/2 for internal communication by default`.
This is crucial change.

Reply:
Done!